api/tests/storage.rs

```diff
@@ -0,0 +1,234 @@
+use near_api_types::{Data, NearToken, StorageBalance};
+
+mod common;
+
+use common::{setup_ft_contract, setup_social_contract, TestContext};
+
+#[tokio::test]
+async fn test_that_generic_account_has_no_storage() {
+    let ctx: TestContext = setup_social_contract().await;
```

Comment:
Consider introducing `rstest` crate to use fixtures that you can easily configure to cache some to be reused between the tests, so running the tests won't spin up a new sandbox per test.